src/app/container/email.service.ts

```diff
@@ -60,8 +29,12 @@ export class EmailService {
    * @param path The path of the tool whose access is being requested
    * @param toolRegistry The tool registry (Quay.io, GitLab, etc) of the tool whose access is being requested
    */
-  private getRequestEmailBody(path: string, toolRegistry: string): string {
-    return encodeURI(`I would like to request access to your Docker image ${path}. ` + `My user name on ${toolRegistry} is <username>`);
+  private static getRequestEmailBody(path: string, toolRegistry: string): string {
+    const imgProvider: string = toolRegistry ? toolRegistry : 'undefined';
+    return encodeURI(
+      `I would like to request access to your Docker image ${path}. ` +
+        `My user name on ${imgProvider} is <put your Docker Hub/quay.io username here>`
```

Comment:
1. `username` instead of `user name` (you have have it as one word later).
2. For `<put your Docker Hub/quay.io username here>`, my suggestion was to actually specify the registry, since we know it, `put your ${imgProvider} username here`.
3. But in discussing it with Gary, he suggested "My username on ${imgProvider} is <username>", so I lean towards that approach.